docs/AVS_EFFECT_ANALYSIS.md

```
@@ -0,0 +1,562 @@
+| `fade` | 1 | 2.0% | Fade effects create smooth color and alpha transitions. Essential for creating smooth visual flow. |
+| `tan(` | 1 | 2.0% | Common AVS function/pattern for visual effects. Provides essential functionality for creating dynamic visualizations. |
+
+## 🎨 Superscope Usage
+
+**Total Unique Superscopes:** 28
+**Total Superscope Instances:** 342
+
+| Superscope Name | Usage Count |
+|----------------|------------|
+| Superscope_36_depth0 | 315 |
+| Quantum Field)
```

markdownlint-cli2 (0.18.1)

Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)